### PR TITLE
[IT] translation week 11

### PR DESCRIPTION
**Pull request by fguiso**
Done: 11.md and 11-1.md
Still to do: 11-2.md and 11-3.md

FYI: the website is displaying this as week 10 because the _config.yml file doesn't include a week 10 section yet

### CODE REVIEW

File: docs/it/week11/11-1.md [hunk 1]
```diff
@@ -0,0 +1,990 @@
+---
+lang: it
+lang-ref: ch.11-1
+lecturer: Yann LeCun
+title: Activation and loss functions (part 1)
```
Comment (Atcold) [inline docs/it/week11/11-1.md:5]: Title translation is missing.

File: docs/it/week11/11-1.md [hunk 2]
```diff
@@ -0,0 +1,990 @@
...
+authors: Haochen Wang, Eunkyung An, Ying Jin, Ningyuan Huang
+date: 13 April 2020
+translation-date: 15 Agosto 2020
+translator: Francesca Guiso
+---
+
+
+<!-- ## [Activation functions](https://www.youtube.com/watch?v=bj1fh3BvqSU&t=15s) -->
```
Comment (Atcold) [inline docs/it/week11/11-1.md:13]: You removed the YouTube link.
Plz, add title translation and missing link.

File: docs/it/week11/11-1.md [hunk 3]
```diff
@@ -0,0 +1,990 @@
+---
+lang: it
+lang-ref: ch.11-1
+lecturer: Yann LeCun
+title: Activation and loss functions (part 1)
+authors: Haochen Wang, Eunkyung An, Ying Jin, Ningyuan Huang
+date: 13 April 2020
+translation-date: 15 Agosto 2020
```
Comment (Atcold) [inline docs/it/week11/11-1.md:8]: `April` -> `Apr`.
`Agosto` -> `Aug`.

File: docs/it/week11/11.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+lang: it
+lang-ref: ch.11
+title: Settimana 11
+translation-date: 15 August 2020
```
Comment (Atcold) [inline docs/it/week11/11.md:5]:
```suggestion
translation-date: 15 Aug 2020
```

**Review by Atcold — CHANGES_REQUESTED** [review]
Great! Just a few edits.

**Comment by fguiso** [comment]
> Great! Just a few edits.

Fixed in the latest commit

**Review by Atcold — APPROVED** [review]
Yay! 🥳

**Pull request merged**